python/testing.rst

```diff
@@ -248,24 +248,24 @@ Inheriting from `lsst.utils.tests.TestCase` rather than `unittest.TestCase` enab
 `lsst.utils.tests.TestCase.assertFloatsNotEqual`
    Asserts that floating point scalars and/or arrays are not equal.
 
-Additionally, :ref:`lsst.afw.coord <pipelines:lsst.afw.coord>`, :ref:`lsst.afw.geom <pipelines:lsst.afw.geom>`, and :ref:`lsst.afw.image <pipelines:lsst.afw.image>` provides additional asserts that get loaded into `lsst.utils.tests.TestCase` when the associated module is loaded.
-These include methods for `Coords`_, `Geom (Angles, Pairs, Boxes)`_, and `Images`_, such as:
+Additionally, :ref:`lsst.afw.geom <pipelines:lsst.afw.geom>` and :ref:`lsst.afw.image <pipelines:lsst.afw.image>` provides additional asserts that get loaded into `lsst.utils.tests.TestCase` when the associated module is loaded.
+These include methods for `Geom (Angles, Pairs, Boxes)`_, and `Images`_, such as:
```

Comment:
```suggestion
These include methods for SpherePoints, `Geom (Angles, Pairs, Boxes)`_, and `Images`_, such as:
```
(Presumably there's some good place to link this to.)

Reply:
Well, they are all linked within the description of each special assert, just a few lines down.

(I also went with SpherePoints inside those brackets, for what it's worth - they're also provided by just lsst.geom, the tricky odd one out is the special assert for WCS)